src/code_80091750.c

```diff
@@ -5596,30 +5597,30 @@ Gfx *func_8009BC9C(Gfx *arg0, MkTexture *arg1, s32 arg2, s32 arg3, s32 arg4, s32
     return arg0;
 }
 
-Gfx *func_8009BEF0(Gfx *arg0, MkTexture *arg1, f32 arg2, f32 arg3, s32 arg4, f32 arg5, f32 arg6) {
+Gfx *print_letter(Gfx *arg0, MkTexture *glyphTexture, f32 arg2, f32 arg3, s32 mode, f32 scale_x, f32 scale_y) {
```

Comment:
scaleX and scaleY

While you're at it. the timer from way before should also be timerBlah instead of timer_blah